application/world/services/auto_bible_generator.py

```
@@ -758,11 +809,73 @@ async def _call_llm_and_parse(self, system_prompt: str, user_prompt: str) -> Dic
 
             content = content.strip()
 
-            # 尝试找到第一个 { 和最后一个 }
+            # 提取 JSON 起点并进行自愈修复
             start = content.find('{')
-            end = content.rfind('}')
-            if start != -1 and end != -1:
-                content = content[start:end+1]
+            if start != -1:
+                # 1. 先尝试正常的截取（容忍前后废话）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)